d2bs/kolbot/libs/bots/IPHunter.js

```diff
@@ -1,26 +1,56 @@
 /**
 *	@filename	IPHunter.js
-*	@author		kolton
+*	@author		kolton, Mercoory
 *	@desc		search for a "hot" IP and stop if the correct server is found
+*	@changes	(2019.10.06) More beeps when IP found; More often movement when IP found (Anti drop measure); Overhead messages with countdown; Logs to D2BS console
 */
 
 function IPHunter() {
-	var ip = Number(me.gameserverip.split(".")[3]);
+    let ip = Number(me.gameserverip.split(".")[3]);
```

Comment:
nit: Doesn't this work like below? And in case gameserverip for some reason is not there, shouldn't there be an if condition to check it like ```if (me.gameserverip)```?

```let ip = +me.gameserverip.split(".")[3];```

Reply:
Can you tell me what the `+` (plus symbol) syntax does and why you think it is more appropriate than `Number(...)`

Reply:
It's just syntactic sugar and can be ignored. But I believe that existence of gameserverip and it having 3 dots should be checked.